_rules/link-skip-block-content-7b576d.md

```diff
@@ -0,0 +1,52 @@
+---
+id: 7b576d
+name: Link for skipping block of content
+rule_type: atomic
+description: |
+  This rule checks that blocks of content can be skipped by a link at their beginning
+accessibility_requirements:
+  wcag-technique:G123: # Adding a link at the beginning of a block of repeated content to go to the end of the block
+		forConformance: false
+		failed: not satisfied
+		passed: further testing needed
+		inapplicable: further testing needed
+input_aspects:
+	- DOM Tree
+	- CSS styling
+	- Language
+authors:
+	- Jean-Yves Moyen
+	- Anne Thyme Nørregard
+---
+
+## Applicability
+
+This rule applies to any [document](#https://www.w3.org/TR/dom/#concept-document) where the [document element](#https://www.w3.org/TR/dom/#document-element) is an HTML `html` element.
+
+## Expectations
+
+For each [section of repeated content](#repeated-content) within the test target, either the last [focusable](#focusable) element before it, or the first [focusable](#focusable) element inside it has a [semantic role](#semantic-role) of `link` and:
+- is [included in the accessibility tree](#included-in-the-accessibility-tree); and
+- is [visible](#visible) when [focused](#focused); and
+- when activated, moves focus immediately after this [section of content](#section-of-content); and

```

Comment:
You need to better define "after this"? Is it to the first element in the focus order, the flat tree, the accessibility tree?